kaggle_environments/agent.py

```diff
@@ -33,7 +33,7 @@ def build_agent(raw, message):
 
     # A URL and will be initialized on the calling server.
     if is_url(raw):
-        def url_agent(o, c, r, i):
+        def url_agent(o, c, r, i, t):
```

Comment:
Timeout needs to be passed by reference as it changes - add the message object back